source/common/umf_helpers.hpp

```diff
@@ -328,6 +351,28 @@ inline umf_result_t createMemoryProvider(
   return UMF_RESULT_SUCCESS;
 }
 
+inline umf_result_t createMemoryProxyPool(umf_memory_provider_handle_t provider,
+                                          umf_memory_pool_handle_t *pool) {
+
+  umf_memory_pool_handle_t _pool = nullptr;
+
+  umf_result_t UmfResult =
+      umfPoolCreate(umfProxyPoolOps(), provider, nullptr, 0, &_pool);
+  UMF_RETURN_UMF_ERROR(UmfResult);
+
+  *pool = _pool;
+
+  return UMF_RESULT_SUCCESS;
+}
+
+inline ur_result_t umfCUDAFree(void *ptr) {
```

Comment:
Yhis is source/common (common for all adapters, not only CUDA). If you want to add CUDA helpers add them to source/adapters/cuda/common.hpp.

Reply:
Removed - replaced with `umfFree()`.